training/cifar/cifar10_deepspeed.py

```diff
@@ -279,6 +280,9 @@ def test(model_engine, testset, local_device, target_dtype, test_batch_size=4):
 def main(args):
     # Initialize DeepSpeed distributed backend.
     deepspeed.init_distributed()
+    _local_rank = int(os.environ.get("LOCAL_RANK"))
+    torch_device = torch.device(f"cuda:{_local_rank}")
+    torch.cuda.set_device(torch_device)

```

Comment:
Please use the device-agnostic accelerator [API](https://www.deepspeed.ai/tutorials/accelerator-abstraction-interface/#write-accelerator-agnostic-models) as follows: 

```suggestion
    get_accelerator().set_device(_local_rank)
```

Reply:
Done, thanks for the suggestion.